src/api/buildSwapTransaction.test.ts

```diff
@@ -283,7 +283,7 @@ describe('buildSwapTransaction', () => {
     expect(sendRequest).toHaveBeenCalledTimes(1);
     expect(sendRequest).toHaveBeenCalledWith(CDP_GET_SWAP_TRADE, [
       {
-        slippagePercentage: '3',
```

Comment:
@0xAlec Has this test been broken? It seems like we've been calling the v1 api with the wrong slippagePercentage (not multiplying the value by 10?)

Reply:
confirmed that this test has been broken